### PR TITLE
Fabo/492 move data calls to lcd client

### PR DESCRIPTION
**Pull request by faboweb**
> Thank you a lot for contributing to Cosmos Voyager!

### Issue

Preparing for #541
closes: #492 

❤️ Thank you!

### CODE REVIEW

File: test/unit/specs/components/staking/__snapshots__/PageDelegates.spec.js.snap [hunk 1]
```diff
@@ -247,7 +247,7 @@ exports[`PageDelegates should filter the delegates 1`] = `
     </div>
   </header>
   <main
-    class="ni-page-main"
+    class="ni-page-main ps"
```
Comment (jolesbi) [inline test/unit/specs/components/staking/__snapshots__/PageDelegates.spec.js.snap]: `ps`?

**Comment by jolesbi** [comment]
tests are going down a tiny - otherwise, this is excellent.

File: test/unit/specs/components/staking/__snapshots__/PageDelegates.spec.js.snap (hunk 1 above)
Comment (faboweb) [reply test/unit/specs/components/staking/__snapshots__/PageDelegates.spec.js.snap]: this is the perfect-scrollbar plugin :/ We need to probably mock it out in tests at some point.

**Comment by faboweb** [comment]
All impacted files have 100% coverage. The total coverage percentage has gone down due to reduced lines of code.

**Pull request merged**